scripts/after_success.sh

```diff
@@ -14,5 +14,5 @@ PACKAGE_JSON_VERSION=$(node -e "console.log(require('./package.json').version)")
 
 if [[ ${COMMIT_MESSAGE} == ${PACKAGE_JSON_VERSION} ]]; then
   echo "running prebuild"
-  node ./node_modules/prebuild/bin.js --all --strip -u ${GITHUB_TOKEN}
+  node ./node_modules/prebuild/bin.js -t 8.0.0 -t 9.0.0 -t 10.0.0 -t 11.0.0 -t 12.0.0 --strip -u ${GITHUB_TOKEN}
```

Comment:
Should we build for "dev" versions after the stable ones are released? (specifically v9 and v11 here)
Also, does `.0.0` act as asterisk (for latest version of that major number) or it really builds for the exact version?

Reply:
I thought about that, especially since node 9 is no longer supported (8 is, under LTS). But I figured that if it works, then there's no sense in really dropping it. 11, while not LTS, is still supported. I'm open to dropping 9 support.

Reply:
It works as an asterisk, essentially. Ever since node 4, I believe, the node ABI is consistent within major versions. So, building for 10.0.0 is the same as for 10.15.3 (i.e. ABI version is 64)

I'm using the values which are required [from here](https://github.com/lgeiger/node-abi#usage)